src/index.ts

```diff
@@ -1,18 +1,6 @@
 import ejs from 'ejs'
 import { helpers, getContext, Property, Parameter } from "@dylibso/xtp-bindgen"
 
-function needsCasting(p: Property | Parameter): boolean {
```

Comment:
I found this ended up being redundant logic and i think i wrote it when i was experiencing a little brain fog